### ISSUE TITLE
HGroupItem does not respect reflowsForAccessibilityTypeSizes in the style 

### ISSUE THREAD

**Issue by thedrick** [airbnb/epoxy-ios#56]
When HGroupItem creates the internal HGroup it should be using the `init(style:)` initializer instead of manually passing in properties. This is causing an issue where setting `reflowsForAccessibilityTypeSizes` in the style does nothing.
